### ISSUE TITLE
Provide "About" screen in Signal

### ISSUE THREAD

**Issue by cascheberg** [WhisperSystems/Signal-Android#4990]
Signal does not provide many information about itself within the app. The only things are the help menu, which links directly to the support page, and the Signal version number hidden in advanced settings. Many apps provide an "About" screen.

I would expect to be able to see:
- the app name, logo and version number
- vendor name and link to website
- some short text regarding the functionality
- hints regarding contributing and funding
- license / open-source information

Basically things that can be found in the [README.md](https://github.com/WhisperSystems/Signal-Android/blob/master/README.md) (but shorter).

**Comment by 2-4601** [comment]
Check out #1356 #1466 #1501 #2053 for duplicates.
